person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  def initialize(age, name = 'Unkonwn', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+  attr_accessor :name, :age
+  attr_reader :id
+
+  private
+
+  def of_age
+    @age >= 18
+  end
```

Comment:
- [ ] The only private method is this `of_age` make ` can_use_services?` public, move it before `of_age` method

Reply:
done as requested